mods/tnt/init.lua

```diff
@@ -404,7 +404,7 @@ minetest.register_node("tnt:gunpowder", {
 		type = "fixed",
 		fixed = {-1/2, -1/2, -1/2, 1/2, -1/2+1/16, 1/2},
 	},
-	groups = {dig_immediate = 2, attached_node = 1, connect_to_raillike = minetest.raillike_group("gunpowder")},
+	groups = {dig_immediate = 2, attached_node = 1, flammable = 5, connect_to_raillike = minetest.raillike_group("gunpowder")},
```

Comment:
Not sure about flammable, it would be weird for it to burn without changing neighbours to 'gunpowder burning'. Perhaps leave it non-flammable while we consider interaction of fire and tnt stuff.